client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/test/checklist-item.tsx

```diff
@@ -48,45 +48,9 @@ describe( 'ChecklistItem', () => {
 
 	describe( 'when a task is incomplete', () => {
 		it( 'hides the task complete icon', () => {
-			render( <ChecklistItem task={ buildTask( { isCompleted: false } ) } /> );
+			render( <ChecklistItem task={ buildTask( { completed: false } ) } /> );
 			const taskCompleteIcon = screen.queryByLabelText( 'Task complete' );
 			expect( taskCompleteIcon ).toBeFalsy();
 		} );
-
-		describe( 'and the task depends on the completion of other tasks', () => {
```

Comment:
These tests are still valid, we've just refactored how we're approaching the problem. I think we should keep them and update the test implementation.

Reply:
@jeyip Sorry for the late reply, I will look into this